src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -1938,6 +1966,251 @@
+  "com.minecolonies.research.magiccompost.name": "Magic Compost",
+  "com.minecolonies.research.master.name": "Master",
+  "com.minecolonies.research.masterbowman.name": "Master Bowman",
+  "com.minecolonies.research.masterswordsman.name": "Master Swordsman",
+  "com.minecolonies.research.memoryaid.name": "Memory Aid",
+  "com.minecolonies.research.morebooks.name": "More Books",
+  "com.minecolonies.research.motherlode.name": "Motherlode",
+  "com.minecolonies.research.multishot.name": "Multishot",
+  "com.minecolonies.research.multishot.subtitle": "Known side effects include double vision.",
+  "com.minecolonies.research.nimble.name": "Nimble",
+  "com.minecolonies.research.nurture.name": "Nurture",
+  "com.minecolonies.research.opera.name": "Opera",
+  "com.minecolonies.research.parry.name": "Parry",
+  "com.minecolonies.research.phd.name": "PhD",
+  "com.minecolonies.research.phd.subtitle": "Not that sort of doctor.",
+  "com.minecolonies.research.provost.name": "Provost",
+  "com.minecolonies.research.puberty.name": "Puberty",
+  "com.minecolonies.research.rapidshot.name": "Rapid Shot",
+  "com.minecolonies.research.recipebook.name": "Recipe Book",
+  "com.minecolonies.research.redstonepowered.name": "Redstone Powered",
+  "com.minecolonies.research.reflective.name": "Reflective",
+  "com.minecolonies.research.reinforced.name": "Reinforced",
+  "com.minecolonies.research.retreat.name": "Retreat",
+  "com.minecolonies.research.richveins.name": "Rich Veins",
+  "com.minecolonies.research.riposte.name": "Riposte",
+  "com.minecolonies.research.rtm.name": "RTM",
+  "com.minecolonies.research.scholarly.name": "Scholarly",
+  "com.minecolonies.research.seemsautomatic.name": "Seems Automatic",
+  "com.minecolonies.research.skills.name": "Skills",
+  "com.minecolonies.research.space.name": "Space",
+  "com.minecolonies.research.spectacle.name": "Spectacle",
+  "com.minecolonies.research.squiretraining.name": "Squire Training",
+  "com.minecolonies.research.steelarmor.name": "Steel Armor",
+  "com.minecolonies.research.steelbracing.name": "Steel Bracing",
+  "com.minecolonies.research.strong.name": "Strong",
+  "com.minecolonies.research.studious.name": "Studious",
+  "com.minecolonies.research.stuffer.name": "Stuffer",
+  "com.minecolonies.research.swift.name": "Swift",
+  "com.minecolonies.research.theater.name": "Theater",
+  "com.minecolonies.research.tools.name": "Tools",
+  "com.minecolonies.research.trickshot.name": "Trick Shot",
+  "com.minecolonies.research.veinminer.name": "Veinminer",
+  "com.minecolonies.research.whatisthisspeed.name": "What Is This Speed?",
+  "com.minecolonies.research.arrowpierce.name": "Arrow Piercing",
+  "com.minecolonies.research.arrowuse.name": "Consume Arrows",
+  "com.minecolonies.research.arrowuse.subtitle": "They work better with ammo.",
+  "com.minecolonies.research.avoidance.name": "Avoidance",
+  "com.minecolonies.research.doubletrouble.name": "Double Trouble",
```

```suggestion
  "com.minecolonies.research.doubletrouble.name": "Double Trouble",
  "com.minecolonies.research.doubletrouble.subtitle": "Double Trouble",
```
Am I using this too much too much?